py/desisim/spec_qa/redshifts.py

```diff
@@ -116,39 +117,82 @@ def calc_stats(simz_tab, objtype, flux_lim=True):
     return stat_dict
 
 
-def load_z(fibermap_files, zbest_files, outfil=None):
+def find_zbest_files(fibermap_data):
+
+    from desimodel.footprint import radec2pix
+    # Init
+    zbest_files = []
+    # Search for zbest files with healpy
+    ra_targ = fibermap_data['RA_TARGET'].data
+    dec_targ = fibermap_data['DEC_TARGET'].data
+    # Getting some NAN in RA/DEC
+    good = np.isfinite(ra_targ) & np.isfinite(dec_targ)
+    pixels = radec2pix(64, ra_targ[good], dec_targ[good])
+    uni_pixels = np.unique(pixels)
+    for uni_pix in uni_pixels:
+        zbest_files.append(desispec.io.findfile('zbest', groupname=uni_pix, nside=64))
+    # Return
+    return zbest_files
+
+'''
+    # Search for zbest files
+    fibermap_data = desispec.io.read_fibermap(fibermap_path)
+    flavor = fibermap_data.meta['FLAVOR']
+    if flavor.lower() in ('arc', 'flat', 'bias'):
+        log.debug('Skipping calibration {} exposure {:08d}'.format(flavor, exposure))
+        continue
+
+    brick_names = set(fibermap_data['BRICKNAME'])
+    import pdb; pdb.set_trace()
+    for brick in brick_names:
+        zbest_path=desispec.io.findfile('zbest', groupname=brick, specprod_dir=args.reduxdir)
+        if os.path.exists(zbest_path):
+            log.debug('Found {}'.format(os.path.basename(zbest_path)))
+            zbest_files.append(zbest_path)
+        else:
+            log.warn('Missing {}'.format(os.path.basename(zbest_path)))
+            #pdb.set_trace()
+'''
```

Comment:
Minor: Let's drop this "commented with quotes" code that searches for zbest files with the old brickname organization that we don't use anymore.

Reply:
removed.

Have added a legend to the summary dict that
should appear towards the top of the YAML file.

I fear the hooks to astropy Table's are deeper than
simply Masked Column.  Would be a rather major
refactor to move to numpy arrays alone.